src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutCompleteContext.php

```diff
@@ -369,6 +369,20 @@ public function iShouldNotBeAbleToProceedCheckoutCompleteStep(): void
         throw new UnexpectedPageException('It should not be possible to complete checkout complete step.');
     }
 
+    /**
+     * @Then I should not be able to confirm order because the :shippingMethod shipping method is not available
+     */
+    public function iShouldNotBeAbleToConfirmOrderBecauseTheShippingMethodIsNotAvailable(ShippingMethodInterface $shippingMethod): void
```

Comment:
Similar as above, there is no need to make a db query to get an entity, as the name will be enough